routergroup.go

```diff
@@ -162,6 +163,21 @@ func (group *RouterGroup) StaticFile(relativePath, filepath string) IRoutes {
 	return group.returnObj()
 }
 
+// StaticFileFS works just like `StaticFile` but a custom `http.FileSystem` can be used instead..
+// router.StaticFileFS("favicon.ico", "./resources/favicon.ico", Dir{".", false})
+// Gin by default user: gin.Dir()
+func (group *RouterGroup) StaticFileFS(relativePath, filepath string, fs http.FileSystem) IRoutes {
```

Comment:
`StaticFileFS` and `StaticFS` have some duplicated codes, we may be improve it.

Reply:
improve StaticFileFS and StaticFile better